server/src/Metrics.hs

```diff
@@ -1,31 +1,27 @@
 module Metrics where
 
 import           Prometheus
+import qualified Network.HTTP.Types as Http
+import qualified Data.ByteString.Char8 as BS8
+
+type RequestCounter = Vector (String, String) Counter
```

Comment:
Looking at this for the first time, I have no idea what this is actually counting. What will the values of the pairs be? 

Try to make the code so that it answers these questions itself alone. If not possible at all, leave a comment.